src/cursor.rs

```diff
@@ -1,247 +1,82 @@
 //! # Cursor
 //!
-//! The `cursor` module provides a functionality to work with the terminal cursor.
+//! The `cursor` module provides functionality to work with the terminal cursor.
 //!
 //! This documentation does not contain a lot of examples. The reason is that it's fairly
 //! obvious how to use this crate. Although, we do provide
 //! [examples](https://github.com/crossterm-rs/examples) repository
 //! to demonstrate the capabilities.
 //!
 //! ## Examples
-//!
-//! Basic usage:
-//!
-//! ```no_run
-//! // You can replace the following line with `use crossterm::TerminalCursor;`
-//! // if you're using the `crossterm` crate with the `cursor` feature enabled.
-//! use crossterm::{Result, TerminalCursor};
-//!
-//! fn main() -> Result<()> {
-//!     // Get a cursor, save position
-//!     let cursor = TerminalCursor::new();
-//!     cursor.save_position()?;
-//!
-//!     // Do something with the cursor
-//!     cursor.goto(10, 10)?;
-//!     cursor.blink(true)?;
-//!
-//!     // Be a good citizen, cleanup
-//!     cursor.blink(false)?;
-//!     cursor.restore_position()
-//! }
-//! ```
-//!
-//! Commands:
+//! Cursor actions can be performed with commands.
```

Comment:
Add empty line after the heading. It's more readable aka the heading is more visible. It also follows already existing documentation style.